Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -118,7 +108,7 @@ class NpmReplCommand : IReplCommand {
             // In case someone copies filename
             string projectDirectoryPath = File.Exists(projectPath) ? Path.GetDirectoryName(projectPath) : projectPath;
             
-            if (!isGlobalCommand && !(Directory.Exists(projectDirectoryPath) && File.Exists(Path.Combine(projectDirectoryPath, "package.json")))) {
+            if (!isGlobalCommand && !Directory.Exists(projectDirectoryPath)) {
```

Comment:
Adding a check for `init` sounds fine for now.

This problem is also related to #578, where someone runs `.npm start` with non terminating apps.